package.json

```diff
@@ -115,7 +115,9 @@
     "electron-compile": "^6.4.2",
     "electron-localshortcut": "^3.0.4",
     "electron-protocol-serve": "1.3.0",
-    "electron-squirrel-startup": "^1.0.0"
+    "electron-squirrel-startup": "^1.0.0",
+    "node-sass": "^4.7.2",
```

Comment:
Since your PR is only translation-related, I'd not commit package.json. Could you please rollback those changes? And package-lock as well, please?